database.env

```diff
@@ -1,3 +1,5 @@
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=mypassword
 POSTGRES_DB=testdb
+ADDRESSES_API_BASE_URL=Test
+ADDRESSES_API_KEY=Test
```

Comment:
These variables are unneccesary here as this file is only being used by the database docker containers. While these two variables are required in the API tests container _(separate container)_.

Reply:
you are right.  I'll remove this.

Reply:
Removed.